internal/services/compute/shared_image_versions_data_source.go

```diff
@@ -45,6 +46,13 @@ func dataSourceSharedImageVersions() *pluginsdk.Resource {
 				Computed: true,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
+						"resource_id": {
```

Comment:
we intentionally don't use the phrase `Resource ID` since it's ambiguous between an Azure Resource ID and a Terraform Resource ID, so for consistency across the Provider this should be:

```suggestion
						"id": {
```